examples/src/main/scala/mhtml/examples/HelloWorld.scala

```diff
@@ -1,4 +1,5 @@
 package mhtml.examples
+import mhtml._
```

Comment:
Wait, mhtml._ import is not necessary anymore since the implicits are defined in the companion objects. Cool! I'll go without the 3 step troubleshooting guide but keep the allowed types quite explicit (e.g., mention Rx AND Var).